apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ApplicationPrintButton/SkillWithExperiences.tsx

```diff
@@ -39,6 +39,25 @@ const PageSection = ({ children }: { children: React.ReactNode }) => (
   </div>
 );
 
+const getRelevantSkillRecord = (
+  experience: Experience,
+  sectionSkill: Skill,
+): string | null => {
+  const experienceSkills = experience.skills ?? [];
+  const applicableSkill = experienceSkills.find(
+    (skill) => skill.id === sectionSkill.id,
+  );
+
+  if (
+    applicableSkill &&
+    applicableSkill.experienceSkillRecord &&
+    applicableSkill.experienceSkillRecord.details
+  ) {
+    return applicableSkill.experienceSkillRecord.details;
+  }
+  return null;
+};
```

Comment:
A better name for this function might be getRelevantSkillRecordDetails

Reply:
Yes, it would definitely be more accurately named for its purpose that way right
https://github.com/GCTC-NTGC/gc-digital-talent/pull/8526/commits/4a499ee8dc987e6a4c379a3ae312769b4fa5ba76